### PR TITLE
Invoice date and translation adjustments

### PR DESCRIPTION
**Pull request by Matt-Yorkley**
#### What? Why?

Addressing #1674. Hub users identified issues with text and date localisation UK printable invoices.

#### What should we test?

Printable invoices when using en-GB

### CODE REVIEW

File: config/locales/en-GB.yml [hunk 1]
```diff
@@ -238,10 +238,20 @@ en-GB:
...
+  invoice_column_unit_price_with_taxes: "Unit price (Incl. tax)"
+  invoice_column_unit_price_without_taxes: "Unit price (Excl. tax)"
+  invoice_column_price_with_taxes: "Total price (Incl. tax)"
+  invoice_column_price_without_taxes: "Total price (Excl. tax)"
+  invoice_column_tax_rate: "Tax rate"
+  tax_invoice: "INVOICE"
+  tax_total: "Total tax (%{rate}):"
+  total_excl_tax: "Total (Excl. tax):"
+  total_incl_tax: "Total (Incl. tax):"
+  invoice_tax_total: "VAT Total:"
```
Comment (enricostano) [inline config/locales/en-GB.yml]: are these keys in `en.yml` too? AFAIK we should only touch `en.yml` to not break Transifex process.

cc/ @mkllnk

**Comment by enricostano** [comment]
BTW... what's the difference between `en_GB.yml` and `en-GB.yml`? I'm just curious.

File: config/locales/en-GB.yml (hunk 1 above)
Comment (Matt-Yorkley) [inline config/locales/en-GB.yml]: Yes, they're in both.

**Comment by Matt-Yorkley** [comment]
The duplicate files are a separate Transifex issue that should be fixed soon...

File: config/locales/en-GB.yml (hunk 1 above)
Comment (enricostano) [inline config/locales/en-GB.yml]: We should use the Transifex process IMHO, if not we'll break it

**Comment by Matt-Yorkley** [comment]
Thanks @mkllnk. I'm not really in charge of the translation or Transifex stuff for the UK, but it's handy to be able to change a few keys like this. I'll take out the `en_GB.yml` changes. Maybe our translation strategy needs a bit of discussion in the UK, but we have it set to pull `en` keys when the `en-GB` keys are missing on production, so I think keeping every key updated isn't a huge issue for us.

**Comment by Matt-Yorkley** [comment]
I've removed the `en_GB.yml` changes now so I think it's ready to go...

**Comment by lin-d-hop** [comment]
@mkllnk Is this ok to be merged for July release as well? Looks like @Matt-Yorkley acted on @enricostano s requested changes :-)

**Comment by mkllnk** [comment]
Yes, I'm planning a final test and merge tomorrow.

**Comment by sstead** [comment]
@mkllnk 
**Testing notes**
- I was told to check that nothing had changed on the invoice in the Aus staging. It's a bit harder to test because we currently have a translation bug in production obscuring the invoice, making comparison difficult.
- From what I can see there are differences in the invoice (see below). From memory, the invoice will be laid out differently for instances that have >1 tax rate than for instances that have just 1. Staging was setup with >1 tax rate and production has 1. So this probably explains the differences in display, but I can't be sure. I tried to delete the second tax rate from staging to confirm this, but it didn't results in a change in the invoice display.
image

As another note, in Aus, we should also have our date displayed as DD-MM-YYYY. Can we implement this on Aus production?

**Comment by sstead** [comment]
Good to go!

**Pull request merged**